src/platform/user/authentication/utilities.js

```diff
@@ -94,9 +93,9 @@ export function verify() {
   return redirect(VERIFY_URL, 'verify-link-clicked');
 }
 
-export function logout() {
+export function logout(version = 'v0') {
```

Comment:
Note the `mfa`, `verify`, and `signup` URLs should all be versioned as well - those requests need to flow through SSOe so that it receives the respond with the new id-proofed or 2fa attributes.

 `mfa` and `verify` are rendered in the "Accounts" page after a user signs in, so will need logic there to pass the version parameter. `signup` is rendered in the sign-in modal

Reply:
@patrickvinograd yeah that makes sense.  I just pushed up a commit, 0fa2231, to address the `signup` urls as well.  I'd like to make a seperate PR for `mfa` and `verify`, as those are being referenced from non React.Component functions and some additional work will need to be done to update.